install/0000_30_machine-api-operator_02_machine.crd.yaml

```diff
@@ -6,31 +6,6 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: machines.machine.openshift.io
 spec:
```

Comment:
why is this being removed?

Reply:
looks like it is the result of the `make gen-crd`, need to check why it happens, probably it was some change under the cluster-api repository